lib/evmone/instructions_calls.cpp

```diff
@@ -95,6 +96,21 @@ Result call_impl(StackTop stack, int64_t gas_left, ExecutionState& state) noexce
     if (has_value && intx::be::load<uint256>(state.host.get_balance(state.msg->recipient)) < value)
         return {EVMC_SUCCESS, gas_left};  // "Light" failure.
 
+    if constexpr (Op == OP_DELEGATECALL)
+    {
+        if (state.rev >= EVMC_CANCUN && is_eof_container(state.original_code))
```

Comment:
Assuming `original_code` is the "caller's code"?

Reply:
Yes. The code DELEGATECALL instruction belongs to.